src/resources/packages/classy/fields/EventLocation/EventLocation.tsx

```diff
@@ -325,69 +325,73 @@ export default function EventLocation( props: FieldProps ) {
 					</div>
 				) }
 
-				{ isAdding && <span className="classy_section-separator"></span> }
-
-				<div className="classy-field__inputs-section classy-field__inputs-section--row">
-					{ ( isAdding || currentVenueIds.length === 0 ) && (
-						<Fragment>
-							<div className="classy-field__input classy-field__input-full-width">
-								<CustomSelectControl
-									__next40pxDefaultSize
-									className="classy-field__control classy-field__control--select"
-									hideLabelFromVision={ true }
-									label={ _x(
-										'Venue selection',
-										'Assistive technology label',
-										'the-events-calendar'
-									) }
-									onChange={ onVenueSelect }
-									options={ options }
-									value={ placeholderOption }
-								/>
-							</div>
+				{ isAdding && currentVenueIds.length > 0 && <span className="classy_section-separator"></span> }
```

Comment:
Since we reference the condition `currentVenueIds.length > 0` multiple times, it might be a good idea to move the condition into a variable for readability.